lightning-invoice/src/lib.rs

```diff
@@ -419,7 +419,7 @@ pub struct InvoiceSignature(pub RecoverableSignature);
 /// The encoded route has to be <1024 5bit characters long (<=639 bytes or <=12 hops)
 ///
 #[derive(Eq, PartialEq, Debug, Clone)]
-pub struct RouteHint(Vec<RouteHintHop>);
+pub struct Route(RouteHint);
```

Comment:
Hmm, I don't think `Route` is right, either, its not a Route in the sense that you can't pay over it, its still a hint. More generally, the bindings will be sad that we have two things called `Route`, even in different scopes. Maybe `InvoiceRouteHint`, if that's not too verbose? I'd maybe vote for changing most of the `route` things in the invoice API to `route_hint` anyway.